assets/js/dprFrontendInit.js

```diff
@@ -1,2 +1,3 @@
-import { initAll } from '/assets/dpr/js/all.mjs'
+import initAll from '../../../../../../../assets/dpr/js/all.mjs'
```

Comment:
Woah, that's crazy. What happened?

Reply:
hmm not sure....

I didn't do that... Perhaps format on save did it